csharp/src/Drivers/BigQuery/BigQueryStatement.cs

```diff
@@ -127,51 +198,57 @@ public override QueryResult ExecuteQuery()
                     }
                 }
             }
+
             ReadSession rs = new ReadSession { Table = table, DataFormat = DataFormat.Arrow };
-            ReadSession rrs = readClient.CreateReadSession("projects/" + results.TableReference.ProjectId, rs, maxStreamCount);
+            ReadSession rrs = clientMgr.ReadClient.CreateReadSession("projects/" + results.TableReference.ProjectId, rs, maxStreamCount);
```

Comment:
Should this have a retry? Given that there's a `CreateReadSessionAsync` I imagine it must be making a network request.